chipsec/cfg/1022/strix.xml

```diff
@@ -0,0 +1,234 @@
+<?xml version="1.0"?>
+<configuration platform="Strix Point">
```

Comment:
The `<sku ... code="Strix" ...>` needs to match the `<configuration platform="Strix Point">`:
https://github.com/chipsec/chipsec/actions/runs/11599009964/job/32297207679?pr=2343

I'm contemplating removing the code attribute on the sku, because it's actually redundant and not needed, but per history, they currently need to match.